docs/specs/code/MULTI_CURRENCY.md

```diff
@@ -0,0 +1,55 @@
+# Abstract
+
+Describe the ability to see and input ADA amount in another currency.
```

Comment:
I think it would be good to be able to see the price at two different moments:

- right now
- at the moment of the transaction

Reply:
Added.